src/main.js

```diff
@@ -170,12 +159,56 @@ function displayNewCover(newCover) {
     }; 
   };
   
-  // Removes a cover when double clicked:
   function deleteCover(id) {
     for(var i = 0; i <= savedCovers.length; i++) {
       if (`${savedCovers[i].id}` === id) {
         savedCovers.splice(i, 1);
         displaySavedCovers();
       };
     };
-  };
\ No newline at end of file
+  };
+
+function checkInput(cover, title, descriptor1, descriptor2) {
+  if(cover && title && descriptor1 && descriptor2) {
+    addToArrays(cover, title, descriptor1, descriptor2);
+    toggleHomeButton();
+    displayNewCover(currentCover);
+  } else {
+    lackOfInputAlert(cover, title, descriptor1, descriptor2)
+  };
+};
+
+function lackOfInputAlert(cover, title, descriptor1, descriptor2) {
+  coverAlert(cover);
+  titleAlert(title);
+  descriptor1Alert(descriptor1);
+  descriptor2Alert(descriptor2);
+};
+
+function coverAlert(cover) {
```

Comment:
We originally had each of these alerts built into one function, but we thought that it was a bit too long. Would love to talk to a mentor to clarify the SRP.